attest/application_key.go

```diff
@@ -56,9 +56,27 @@ func (s *signer) Public() crypto.PublicKey {
 	return s.pub
 }
 
-// KeyConfig encapsulates parameters for minting keys. This type is defined
-// now (despite being empty) for future interface compatibility.
+// Algorithm indicates an asymmetric algorithm to be used.
+type Algorithm string
+
+// Algorithm types supported.
+const (
+	EC  Algorithm = "EC"
```

Comment:
nit: I think this should be "ECDSA"

Reply:
I wanted to keep it consistent with certtostore but indeed ECDSA is more precise. Done.